src/components/sidebar/components/sidebar-nav-menu-item/index.tsx

```diff
@@ -150,6 +151,26 @@ const SidebarNavLinkItem = ({ item }: SidebarNavLinkItemProps) => {
 	}
 }
 
+/**
+ * Handles rendering a button and icon for the sidebar.
+ * Currently used for a 'sign out' action on the profile page
+ */
+export function SidebarNavMenuButton({ item }: SidebarNavMenuButtonProps) {
```

Comment:
Created a new component here to handle the `Sign out` button in the sidebar. This is created to support the needs of this usecase, I figure we can expand / abstract as further usecases for this crop up.

Reply:
Perfect place to put this! 🙌🏻 Totally agree with this approach.